revscoring/features/diff.py

```diff
@@ -31,17 +31,73 @@ def process_chars_added(diff_added_segments):
 
 chars_added = Feature("diff.chars_added", process_chars_added,
                       returns=int, depends_on=[diff.added_segments])
+"""
+Represents number of characters added in this edit.
+
+:Returns:
+    int
+
+:Example:
+    ..code_block :: python
```

Comment:
Should be "..code-block:: python".  See http://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#code-block-directive

Sorry.  That's my fault for typing it wrong when I showed it to you.